pyro/infer/svi.py

```diff
@@ -128,4 +132,8 @@ def step(self, *args, **kwargs):
         # zero gradients
         pyro.infer.util.zero_grads(params)
 
-        return torch_item(loss)
+        # RWS: handle loss = (wake_theta_loss, wake_phi_loss)
```

Comment:
I think it would be cleaner to (1) handle arbitrary `tuple` and `namedtuple` objects
```py
if isinstance(loss, tuple):  # including namedtuple
    return type(loss)(map(torch_item, loss))
else:
    return torch_item(loss)
```
and (2) move this logic into `torch_item` and `torch_isnan` so that svi.py is unmodified. WDYT?

Reply:
Completely fine with whatever the best-fit approach for pyro ought to be. We just went with this approach to minimise edits. :)

Reply:
On second thought I guess this change is fine here. It would be nice to use the more general form though, `type(loss)(map(torch_item, loss))`.

Reply:
Done in `0a48f5c8`.